src/bdscript/removeAllComponents.md

```diff
@@ -0,0 +1,26 @@
+# $removeAllComponents
+Removes all components from a message.
+
+> **Note** : Components are [buttons](../guides/buttons.md) and [select menus](../guides/selectmenu.md).
+
+## Usages
```

Comment:
Instead of usages, split this file in two.
You can use `$boostCount` as an example.
See [`$boostCount`](https://github.com/NilPointer-Software/bdfd-wiki/blob/dev/src/bdscript/boostCount.md) and [`$boostCount[]`](https://github.com/NilPointer-Software/bdfd-wiki/blob/dev/src/bdscript/boostCountComplex.md)

Reply:
Alright, done!